alchemy-web/docs/providers/sentry/client-key.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Managing Sentry Client Keys with Alchemy
+description: Learn how to create, configure, and manage Sentry client keys using Alchemy.
+---
+
+# Sentry ClientKey
+
+Create and manage Sentry client keys.
+
+## Authentication
+
+You can authenticate with Sentry in two ways:
+
+1. Environment variable (recommended):
+   ```bash
+   # .env
+   SENTRY_AUTH_TOKEN=your_auth_token
+   ```
+
+2. Pass the token directly:
+   ```typescript
+   const key = await ClientKey("my-key", {
+     name: "My Key",
+     project: "my-project",
+     organization: "my-org",
+     authToken: "your_auth_token"
```

Comment:
should be `alchemy.secret("your_auth_token")`

Reply:
Resolved via 274939a5c3123e1003bcdb36d106ad9eaca431af